src/index.html

```diff
@@ -1,14 +1,35 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="navigation">
+        <img class="box box-0"
+            src="images\logo.png"
+          alt="Moyo"
+          >
+        <a class="box box-1 is-active">APPLE</a>
+        <a class="box box-2">SUMSUNG</a>
+        <a class="box box-3">СМАТРФОНЫ И ТЕЛЕФОНЫ</a>
+        <a class="box box-4" data-qa="hover">НОУТБУКИ И КОМПЬЮТЕРЫ</a>
+        <a class="box box-5">ГАДЖЕТЫ</a>
+        <a class="box box-6">ПЛАНШЕТЫ</a>
+        <a class="box box-7">ФОТО</a>
+        <a class="box box-8">ВИДЕО</a>
```

Comment:
```suggestion
        <a class="link is-active">APPLE</a>
        <a class="link ">SUMSUNG</a>
        <a class="link ">СМАТРФОНЫ И ТЕЛЕФОНЫ</a>
        <a class="link " data-qa="hover">НОУТБУКИ И КОМПЬЮТЕРЫ</a>
        <a class="link ">ГАДЖЕТЫ</a>
        <a class="link ">ПЛАНШЕТЫ</a>
        <a class="link ">ФОТО</a>
        <a class="link ">видео</a>
```
css -> text-transform uppercase